rest_framework/validators.py

```diff
@@ -211,7 +214,9 @@ def __call__(self, attrs):
         queryset = self.exclude_current_instance(attrs, queryset)
         if queryset.exists():
             message = self.message.format(date_field=self.date_field)
-            raise ValidationError({self.field: message})
+            raise ValidationError({
+                self.field: message,
+            })
```

Comment:
Can you explain why `ValidationError` is taking a dict here and not a simple message ? It is really not clear to me why, even by looking at the code base.